I/igraph/build_tarballs.jl

```diff
@@ -0,0 +1,80 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "igraph"
+version = v"0.10.15"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/igraph/igraph.git", "635b432eff0a89580ac9bb98068d2fbc8ef374f2")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+
+cd $WORKSPACE/srcdir/igraph/
+
+
+if [[ "${target}" == *mingw* ]]; then
+    LBT=blastrampoline-5
+else
+    LBT=blastrampoline
+fi
+
+if [[ "${target}" == *apple* ]]; then
+    LTO=OFF
+else
+    LTO=ON
+fi
```

Comment:
LTO works fine with Apple's compiler, and we generally recommend using LTO. You can set `-DIGRAPH_ENABLE_LTO=AUTO` to use LTO whenever available. This setting used to fail only with some older MinGW, but I believe it no longer does.

Reply:
noted, thanks!